### ISSUE TITLE
Feature Request: Video Stream with Separate AP

### ISSUE THREAD

**Issue by alecGraves** [dji-sdk/Tello-Python#26]
The limitation is mentioned in [the FAQ](https://github.com/dji-sdk/Tello-Python/blob/693776d65b691525773adf4f320f034946fb15b2/TelloPython_FAQ.txt): 
```
Q8:Can Tello EDU transmit video streams externally in station mode (send ap 'ssid' 'password' command for Tello EDU)?

A8:This feature is currently not supported.
```

Can this feature be added to the Firmware / SDK? It would make the EDU edition much more useful for may lab. 

Currently, this limitation removes all benefits of having the EDU edition because I need live video from all drones in the swarm.

**Comment by BRNKR** [comment]
I agree. I thought this would be possible. That is the reason why we bought three of them.

**Comment by spacemandev-git** [comment]
Came here to make this issue +1

**Comment by alecGraves** [comment]
As a kind of work around, you can use multiple usb wi-fi adapters and some linux networking to recieve multiple video streams and control multiple Tellos from a single computer. And it works with both EDU and regular edition!

See my fork for implementation:
[Check out the Diff](https://github.com/alecGraves/Tello-Python/commit/37ace18a11e8e1692e0a5fefd9dc7365e659ed81)

And the interface string is just the shortened name of the wifi device (e.g. when you type `ip a` or `ifconfig` in the linux terminal). Usually it is things like 'wlan0', but my dongle had the weird one you see in the diff.
